collector/container/konflux.Dockerfile

```diff
@@ -96,34 +96,24 @@ RUN ./builder/install/install-dependencies.sh && \
     ctest -V --test-dir ${CMAKE_BUILD_DIR} && \
     strip -v --strip-unneeded "${CMAKE_BUILD_DIR}/collector/collector"
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS support-packages-downloader
-
-WORKDIR /staging
-
-COPY kernel-modules/MODULE_VERSION kernel-modules/MODULE_VERSION
-COPY collector/container/scripts/download-support-package.sh download-support-package.sh
-
-RUN ./download-support-package.sh
-
-# Do NOT use follow_tag here, as we do not need or want collector to be rebuilt
-# with each drivers build (which may become very frequent)
+# 0.1.0 is a floating tag and it's used intentionally to pick up the most recent downstream drivers build without
+# having to routinely and frequently bump tags here.
 FROM brew.registry.redhat.io/rh-osbs/rhacs-drivers-build-rhel8:0.1.0 AS drivers-build
-# FROM registry-proxy.engineering.redhat.com/rh-osbs/rhacs-drivers-build-rhel8:0.1.0 AS drivers-build
 
 # TODO(ROX-20312): we can't pin image tag or digest because currently there's no mechanism to auto-update that.
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS unpacker
 
 RUN microdnf install -y unzip findutils
 WORKDIR /staging
 
-COPY --from=support-packages-downloader /staging/support-pkg.zip /staging/
+COPY staging/support-pkg.zip /staging/
```

Comment:
Would it be more straightforward to just copy `staging` (with everything inside) to `/` (so that the resulting paths are `/staging/...`)?

Asking because this `COPY staging/support-pkg.zip /staging/` looks a bit odd but I don't really know if my suggestion would look better.

Reply:
Even though it creates a coupling between the download script and the Dockerfile based on the filename, I prefer to be explicit here to avoid copying other contents from `staging`. There is none today, but to future proof the script.